chain/jsonrpc/src/lib.rs

```diff
@@ -916,6 +919,41 @@ impl JsonRpcHandler {
         Ok(near_jsonrpc_primitives::types::chunks::RpcChunkResponse { chunk_view })
     }
 
+    async fn congestion_info(
+        &self,
+        request_data: near_jsonrpc_primitives::types::congestion::RpcCongestionLevelRequest,
+    ) -> Result<
+        near_jsonrpc_primitives::types::congestion::RpcCongestionLevelResponse,
+        near_jsonrpc_primitives::types::congestion::RpcCongestionLevelError,
+    > {
```

Comment:
Gosh I hate those fully qualified names. How would you feel about breaking that convention? :)

Reply:
I vouch against it. While I feel your concerns, it is much easier to reason about what's happening in the piece of code without jumping here and there to understand where this structure goes from fully. This convention is ugly yet helpful

Reply:
I don't see your point, those structs are unique and local in this context. I understand the need for full qualification where there are multiple structs with the same names but this is not the case here. Additionally any modern IDE will show you the full name if you just hover over it. The general consensus in majority of our codebase (and most other codebases AFAIK) is to not use fully qualified names unless there is good reason for it. Is there something different about json rpc to justify it?

Reply:
Nothing special about json rpc. When we were actively working on this crate we made it like this. I am not a fan of mixing the import styles in a crate and don't think we want to make this change. I have plans for refactoring here. And then it will be appropriate to change the style here to follow the pattern you use in other crates in nearcore. Sounds good?

Reply:
> I am not a fan of mixing the import styles in a crate a 

Yeah that's fair enough, I can't argue with sticking to convention :) 

> I have plans for refactoring here. And then it will be appropriate to change the style here to follow the pattern you use in other crates in nearcore. 

Yeah that sounds great thank you!

Reply:
My two cents: I don't like the fully qualified names either but it's a matter of opinion, not something that's strictly worse.
If such a refactor happens in the future for the crate in question, I'm not opposed to it. But I wouldn't put a high priority on it. I'm generally sceptical of purely opinion based refactors, since those can go in circles if someone else "improves" the code in the opposite direction, while completely trashing the git blame history in the process.